docs/local-deploy.md

```diff
@@ -0,0 +1,58 @@
+
+# How to deploy your foam locally (using jekyll)
+
+Recipe for building your GitHub Pages site locally to preview and test changes to your site.
+
+! **Disclaimer**: This Recipe is more suited for "tech savy" foam users since it requires the usage of the command line.
+
+# Prerequisites
+- Install [Jekyll](https://jekyllrb.com/docs/installation/)
+
+    Github [recommends](https://docs.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll) using Bundler to install and run Jekyll. Bundler manages Ruby gem dependencies, reduces Jekyll build errors, and prevents environment-related bugs. 
+    
+    To install Bundler:
+
+    - Install Ruby (https://www.ruby-lang.org/en/documentation/installation/)
+    - Install Bundler: `gem install bundler`
+
+# Instructions
+- Open the command line
+- Navigate into your foam folder
+- Switch to the github pages branch (if you do not have one create it):
```

Comment:
I think we can skip this step. As it is for tech-savvy users, if they want to create a new branch to do the changes in then that's up to them.

Reply:
Good point! - removed